templates/minimal/README.md

```diff
@@ -81,15 +95,18 @@ be used to run the minimal template's runtime. `polkadot-omni-node` binary crate
 
 Please see installation section on [crates.io/omni-node](https://crates.io/crates/polkadot-omni-node).
 
-#### Build `minimal-template-runtime`
+#### Build `minimal-template-runtime` (A reminder to  build on the most updated version of polkadot-sdk master)
```

Comment:
We can remove the reminder addition. We still waant people to refer to the dedicated minimal-template repository, so building out of `polkadot-sdk` can be done, but isn't the user flow we're looking. This readme will end up in this minimal-template dedicated repo, so we shouldn't refer back to `polkadot-sdk` from there.